### ISSUE TITLE
A couple of issues/suggestions

### ISSUE THREAD

**Issue by FRBFStudios** [Weebifying/ReDash-geode#2]
First, the suggestions: 
1. Right now, there is no way of finding out whether or not the daily/weekly level has been completed except for clicking it and looking at the percentage bar on the level page. I would really appreciate if there was better indication for this (such as replacing the daily level with the timer like in the vanilla game, or, what would be the much more clean and practical option IMO, something like replacing the reward with a checkmark.
2. Would it be possible to make this mod and Restart Buttons compatible in a way where these buttons are still in the top left corner instead on the bar on the right side? Right now, you even have to scroll down towards them, which kind of defeats the purpose of them being a practical, quickly accessable feature.

There are also some bugs/crashes that are really annoying:
1. A crash that has a 50/50 chance of occuring when claiming a quest on my Windows 11 PC. I already reported it to the Geode Discord Server and they told me that Overcharged Main Menu was most likely the mod causing it. I can send the crashlog in a few hours because right now I can't access my PC.
2. Incompatibility with the new Daily Streak mod so the streak counter just isn't displayed (however, I've already reported this to that mod's GitHub so they are propably going to fix it already).
3. The dynamic texts on the menu, such as your own leaderboard placement, the amount of saved and created levels, the star counter for paths and even the exclamation marks above the quest- and daily chest buttons only update very rarely. So far the only reliable way for me to update them was going into the icon kit and exiting it.

Maybe I forgot something, but these are already a lot of complaints and I would already be happy if you fixed the crash, so I will end this report here. If you looked into any of this I would really appreciate it :)

**Comment by FRBFStudios** [comment]
Crashlogs:
[2024-08-10_17-11-31.txt](https://github.com/user-attachments/files/16573425/2024-08-10_17-11-31.txt)
[2024-08-10_17-15-21.txt](https://github.com/user-attachments/files/16573426/2024-08-10_17-15-21.txt)
[2024-08-10_17-44-12.txt](https://github.com/user-attachments/files/16573427/2024-08-10_17-44-12.txt)
[2024-08-11_12-58-39.txt](https://github.com/user-attachments/files/16573428/2024-08-11_12-58-39.txt)

**Comment by HicalebiH** [comment]
I think the crash might be 100% chance, happening to me and couldn't claim a quest until I disabled the mod-

**Comment by FRBFStudios** [comment]
Yes but iirc it didn't happen the first few days? Maybe an incompatibility

**Comment by FRBFStudios** [comment]
Please look into this :( it has been a month and I really like this mod

**Comment by su226** [comment]
This problem still exists in latest version, please fix it.
Log: [Geode 2025-05-30 16.18.37.log](https://github.com/user-attachments/files/20517858/Geode.2025-05-30.16.18.37.log)
Crash Log: [2025-05-30_16-18-43.log](https://github.com/user-attachments/files/20517865/2025-05-30_16-18-43.log)
